src/components/PrintModal/PrintModal.js

```diff
@@ -190,23 +191,24 @@ class PrintModal extends React.PureComponent {
       const pageIndex = pageNumber - 1;
       const zoom = 1;
       const printRotation = this.getPrintRotation(pageIndex);
-      const onCanvasLoaded = canvas => {
+      const onCanvasLoaded = async canvas => {
         this.pendingCanvases = this.pendingCanvases.filter(
           pendingCanvas => pendingCanvas !== id,
         );
         this.positionCanvas(canvas, pageIndex);
-        this.drawAnnotationsOnCanvas(canvas, pageNumber).then(() => {
-          const img = document.createElement('img');
-          img.src = canvas.toDataURL();
-          img.onload = () => {
-            this.setState(({ count }) => ({
-              count: count < 0 ? -1 : count + 1,
-            }));
-            resolve(img);
-          };
-        });
-      };
 
+        if (this.state.includeAnnotations) {
+          await this.drawAnnotationsOnCanvas(canvas, pageNumber);
```

Comment:
The `drawAnnotationsOnCanvas` function actually does two things. It draws all the markup annotations, such as highlight, comment, freehand, etc on the canvas, and it also draws all the widget annotations on it. I think when `includeAnnotations` is not checked, we don't want to see markup annotations, but still want to see widget annotations. Can you check with Sardor(the trello card creater) to see if this is the case?

If this is true, then you probably need to tweak the logic a bit here, since currently if the checkbox is unchecked, widget annotations don't appear in the print document. Let me know if you have any questions. Thanks!

Reply:
Sardor confirmed that
> this is option basically to print PDF without all markup annotations and widgets

Reply:
Thanks for the info. I talked to Sardor and read the discussion in the ticket. It seems that the client didn't specifically say about widget annotations, so that means we can have more flexibility here.

Personally I think we should still keep widget annotations even if it's unchecked, but I will leave the decision to Matt.

@mparizeau-pdftron Can you take a look at the discussion and share your thoughts when you have time? Thanks!

Reply:
Yeah I agree let's keep widgets when this option is checked

Reply:
sounds good, I will modify the code.